share/metainfo/net.lutris.Lutris.metainfo.xml

```diff
@@ -3,21 +3,21 @@
 <component type="desktop">
   <id>net.lutris.Lutris</id>
   <project_license>GPL-3.0</project_license>
-  <metadata_license>GPL-3.0</metadata_license>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
If you change this, please change it accordingly in `debian/copyright`.

Reply:
I don't know why this was changed. I don't want licenses other than GPL-3 in the project.